app/main/session_parser.py

```diff
@@ -288,6 +289,65 @@ def get_iSpindel_graph_data(chart_id, voltage, session_data):
     return graph_data
 
 
+def load_tilt_session(file):
+    info = file.stem.split('#')
+    
+    # 0 = Date, 1 = Device UID
+    json_data = load_session_file(file)
+    
+    chart_id = info[0] + '_' + str(info[1])
+    alias = info[1] if info[1] not in active_tilt_sessions else active_tilt_sessions[info[1]].alias
+    
+    # filename datetime string format "20200615_205946"
+    server_start_datetime = datetime.strptime(info[0], '%Y%m%d_%H%M%S')
+    # json datetime `timeStr` format "2020-06-15T20:59:46.280731" (UTC) ; 'time' milliseconds from epoch
+    server_end_datetime = datetime.strptime(info[0], '%Y%m%d_%H%M%S')
+    if len(json_data) > 0:
+        # set server start datetime to the first data log entry (approx -1hr from session file creation)
+        server_start_datetime = epoch_millis_converter(json_data[0]['time'])
+
+        # set server end datetime to last data log entry
+        server_end_datetime = epoch_millis_converter(json_data[-1]['time'])
+
+    return ({
+        'uid': info[1],
+        'filename': Path(file).name,
+        'filepath': Path(file),
+        'alias': alias,
+        'date': server_start_datetime,
+        'end_date': server_end_datetime,
+        'name': alias,  # should change to brew/user defined session name
+        'data': json_data,
+        'graph': get_tilt_graph_data(chart_id, None, json_data)
+    })
+
+
+def get_tilt_graph_data(chart_id, voltage, session_data):
+    temp_data = []
+    gravity_data = []
+    for data in session_data:
+        temp_data.append([data['time'], float(data['temp'])])
+        gravity_data.append([data['time'], float(data['gravity'])])
+    graph_data = {
+        'chart_id': str(chart_id),
+        'title': {'text': 'Fermentation'},
+        'series': [
+            {
+                'name': 'Temperature',
+                'data': temp_data
+            }, {
+                'name': 'Specific Gravity',
+                'data': gravity_data,
+                'yAxis': 1
+            }
+        ],
+    }
+
+    if voltage:
+        graph_data.update({'subtitle': {'text': 'Voltage: ' + voltage}})
+    return graph_data
```

Comment:
Does tilt broadcast the voltage? I don't think so. We should change this and instead for the subtitle of the graph show the `color` of the device (eg. BLACK, BLUE, PINK, GREEN, etc) where in Picoferm we display voltage.

Reply:
Did not see this. I chose to show the RSSI here since i knew voltage was not supported.  

For the color, if you think it's important, I was thinking we could out it in parens next to the name if it has not beed added

F0B45560-D29D-4717-B89A-2E6F9B9EA010 (Black)

Reply:
i looked at this but it will need some plumbing. i think it can come later if needed. I'd rather put the effort into making it easier to "add" new devices than it is now.